.github/CODEOWNERS

```diff
@@ -1,3 +1,4 @@
 # Define code owners (individuals or teams that are responsible for code in this repository)
 # More about code owners at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 *       @ConduitIO/conduit-core
```

Comment:
This CODEOWNERS file is actually meant for the template repo itself. Every time we open a PR here (like this one) will ask a review from the conduit-core team, so we still need it here. In [`setup.sh`](https://github.com/ConduitIO/conduit-connector-template/blob/3978d55181a0371cc3855b3176ee5a8e0cd8e481/setup.sh#L30) we even remove this line, because the assumption is that the developer will set up their own file if needed. So I'm not sure how to make this better - maybe something in `setup.sh` that asks the user about the codeowners?